website_sale_product_minimal_price/models/product_template.py

```diff
@@ -73,7 +73,7 @@ def _get_cheapest_info(self, pricelist):
             for qty in [1, 99999999]:
                 product_price = product.with_context(
                     quantity=qty, pricelist=pricelist.id
-                ).price
+                ).lst_price
```

Comment:
`_get_product_price` probably fits better the needed migration value: https://github.com/odoo/odoo/blob/4d119ea9fafd25ed4709eb868132b28a54a41bc0/addons/product/models/product_pricelist.py#L76

Check the migration commit for this field: https://github.com/odoo/odoo/commit/9e99a9df464d97a74ca320d200599f7dce2d3050

Reply:
Okey, thanks